be/src/olap/schema_change.cpp

```diff
@@ -1065,48 +1049,25 @@ Status SchemaChangeDirectly::process(RowsetReaderSharedPtr rowset_reader,
         return Status::OLAPInternalError(OLAP_ERR_ALTER_STATUS_ERR);
     }
 
-    // rows filtered by zone map against delete handler
-    add_filtered_rows(rowset_reader->filtered_rows());
-
-    // Check row num changes
-    if (config::row_nums_check) {
```

Comment:
Why do we skip this check? Is this redundant or useless?

Reply:
I move those common logic to `SchemaChange::process`, and some specific logic is placed in `_inner_process`